modules/installation-and-upgrade/pages/db-migration-xy.adoc

```diff
@@ -4,11 +4,14 @@
 This section covers upgrading the PostgreSQL database to the latest version.
 If you are already using PostgreSQL 14, you do not need to perform this migration.
 
-If you want to upgrade to the latest {productname} version, you must be using PostgreSQL version 13 or 14, depending on the underlying operating system:
+If you want to upgrade to the latest {productname} version, you must be using PostgreSQL version 13, 14, or 15 depending on the underlying operating system:
 
 * If you are running SLES 15 SP4, use PostgreSQL 14.
 * If you are running {opensuse} Leap 15.4, use PostgreSQL 14.
 
+* If you are running SLES 15 SP5, use PostgreSQL 15.
```

Comment:
```suggestion
* If you are running SUSE Linux Enterprise Server 15 SP5, use PostgreSQL 15.
```

Reply:
There is a reason we have attribute. :) Woops!